src/khoj/routers/api_chat.py

```diff
@@ -219,6 +220,9 @@ async def chat(
     stream: Optional[bool] = False,
     slug: Optional[str] = None,
     conversation_id: Optional[int] = None,
+    city: Optional[str] = None,
+    region: Optional[str] = None,
+    country: Optional[str] = None,
```

Comment:
Should we just pass a single location parameter from the clients? It can be set to `location = city, region, country`?

Reply:
I think it's better to keep separate as some clients wouldn't have access to all three parameters. For example, the WhatsApp client would only potentially have access to `country`. I think it makes the format easier for the clients to have each param separate.